metricbeat/module/system/system.go

```diff
@@ -53,5 +59,16 @@ func NewModule(base mb.BaseModule) (mb.Module, error) {
 		initModule()
 	})
 
-	return &Module{BaseModule: base, HostFS: *HostFS, IsAgent: fleetmode.Enabled()}, nil
+	config := Config{
+		HostFS: "",
+	}
+	err := base.UnpackConfig(&config)
+	if err != nil {
+		return nil, err
+	}
+	if *HostFS != "" {
+		config.HostFS = *HostFS
+	}
+
+	return &Module{BaseModule: base, HostFS: config.HostFS, IsAgent: fleetmode.Enabled()}, nil
```

Comment:
We're probably gonna have to work a little harder to merge the values from the beat config and the CLI. First of all, `system_linux.go` will need to be changed to grab its value from somewhere else, presumably the merged config. Secondly, we might want to be a little more clear about what value takes precedence, since one value can override another here. Maybe some logic for `if *HostFS != "" && config.HostFS != "" { error() }` or something like that? Just worried about one implicitly overriding the other.